builtin/game/chatcommands.lua

```diff
@@ -71,7 +71,7 @@ end
 --
 core.register_chatcommand("me", {
 	params = "<action>",
-	description = "chat action (eg. /me orders a pizza)",
+	description = "Display chat action (eg. /me orders a pizza)",
```

Comment:
Maybe describe `"Chat about yourself in the 3rd person (eg. /me orders a pizza)"`?
There shouldn't be written what the example does, the player should just try it out and he will see what it does, the additional text would be irritating.